src/cockpit/channel.py

```diff
@@ -386,6 +386,7 @@ async def create_transport(self, loop: asyncio.AbstractEventLoop, options: JsonO
         raise NotImplementedError
 
     def do_open(self, options: JsonObject) -> None:
+        self._transport = None
```

Comment:
This is the wrong spot for that.  Please move this to an initialization at the class level where all the other variables are initialized.  I'm not sure why I missed this one...

Reply:
You know I despise instance variables that pretend to be class variables. *That* be the wrong spot IMO. If you insist I'll do it to get it out of the way, but this really belongs into the constructor (and open() is kinda sorta the constructor for Channels).